.github/workflows/ci.yml

```
@@ -271,6 +271,66 @@ jobs:
         env:
           SEMGREP_APP_TOKEN: ${{ secrets.SEMGREP_APP_TOKEN }}
 
+      - name: Typosquat detection (new npm dependencies)
+        if: needs.changes.outputs.dapp == 'true' || needs.changes.outputs.shared == 'true'
+        run: |
+          # Check for typosquatting patterns in package.json dependencies
+          # Uses npm package name similarity to detect potential typosquats
+          python3 -c "
+import json, sys, re
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)